### PR TITLE
rename EventProcessor* to RoundManager*

### PR DESCRIPTION
**Pull request by haroldcarr**
renamed the type, fields, lens

run-everything passes

### CODE REVIEW

File: LibraBFT/Impl/Properties/VotesOnceDirect.agda [hunk 1]
```diff
@@ -78,11 +78,11 @@ module LibraBFT.Impl.Properties.VotesOnceDirect where
 
   votesOnce₁ : VO.ImplObligation₁
   votesOnce₁ {pid' = pid'} r (step-msg {_ , P m} _ psI) {v' = v'} {m' = m'}
-             pkH v⊂m (here refl) sv ¬msb vspkv v'⊂m' m'∈pool sv' ep≡ r≡
```
Comment (mark-moir) [inline LibraBFT/Impl/Properties/VotesOnceDirect.agda:81]: These changes should be reverted (see other comment about `ValidSenderForPK⇒ep≡`).  Also, maybe change `ep≡` to `eid≡` to be more clear.

File: LibraBFT/Impl/Properties/VotesOnceDirect.agda [hunk 2]
```diff
@@ -39,29 +39,29 @@ module LibraBFT.Impl.Properties.VotesOnceDirect where
...
                    → MsgWithSig∈ pk (ver-signature sig) (msgPool pre)
                    → ValidSenderForPK (availEpochs pre) v pid pk
-                   → (₋epEC (peerStates pre pid)) ^∙ epEpoch ≡ (v ^∙ vEpoch)
-                   → v ^∙ vRound ≤ (₋epEC (peerStates pre pid)) ^∙ epLastVotedRound
-  oldVoteRound≤lvr (step-s r (step-epoch _)) pidIn pkH sig msv vspk ep≡ = {!!}
+                   → (₋rmEC (peerStates pre pid)) ^∙ rmEpoch ≡ (v ^∙ vEpoch)
+                   → v ^∙ vRound ≤ (₋rmEC (peerStates pre pid)) ^∙ rmLastVotedRound
+  oldVoteRound≤lvr (step-s r (step-epoch _)) pidIn pkH sig msv vspk rm≡ = {!!}
   oldVoteRound≤lvr {pid = pid'} {pre = pre} (step-s r (step-peer {pid = pid} cheat@(step-cheat f c)))
-                    pidIn pkH sig msv vspk ep≡
```
Comment (mark-moir) [inline LibraBFT/Impl/Properties/VotesOnceDirect.agda:46]: See other comments about `ValidSenderForPK⇒ep≡`.  Please change all of these `rm≡`s to `eid≡`.

File: LibraBFT/Impl/Handle.agda [hunk 3]
```diff
@@ -21,21 +21,21 @@ module LibraBFT.Impl.Handle
...
+ open import LibraBFT.Impl.Consensus.ChainedBFT.RoundManager hash hash-cr
  open RWST-do
 
- -- This represents an uninitialised EventProcessor, about which we know nothing, which we use as
- -- the initial EventProcessor for every peer until it is initialised.
+ -- This represents an uninitialised RoundManager, about which we know nothing, which we use as
+ -- the initial RoundManager for every peer until it is initialised.
  postulate
-   fakeEP : EventProcessor
+   fakeEP : RoundManager
```
Comment (mark-moir) [inline LibraBFT/Impl/Handle.agda:30]: Should become `fakeRM`.

File: LibraBFT/Impl/Consensus/Types.agda [hunk 4]
```diff
@@ -36,90 +36,90 @@ module LibraBFT.Impl.Consensus.Types where
...
      -- If we want to add pieces that neither contribute to the
      -- construction of the EC nor need one, they should be defined in
-     -- EventProcessor directly
-  open EventProcessor public
+     -- RoundManager directly
+  open RoundManager public
 
-  α-EC-EP : EventProcessor → EpochConfig
-  α-EC-EP ep = α-EC ((₋epEC ep) , (₋epEC-correct ep))
+  α-EC-EP : RoundManager → EpochConfig
```
Comment (mark-moir) [inline LibraBFT/Impl/Consensus/Types.agda:108]: Should become `α-EC-RM`.

**Review by mark-moir — CHANGES_REQUESTED** [review]
Thanks @haroldcarr!  I have a few minor comments, and I'll be happy to merge after you address them.  Please remember to "signoff" your last commit (see the [Contribution Guide](https://github.com/oracle/bft-consensus-agda/blob/main/CONTRIBUTING.md#contributing-code)).

File: LibraBFT/Impl/Consensus/ChainedBFT/RoundManager.agda [hunk 5]
```diff
@@ -18,11 +18,11 @@ open import LibraBFT.Abstract.Types.EpochConfig UID NodeId
 
 -- This is a minimal/fake example handler that obeys the VotesOnce rule, enabling us to start
 -- exploring how we express the algorithm and prove properties about it.  It simply sends a vote for
--- 1 + its LatestVotedRound, and increments its LatestVotedRound.  It is called EventProcessor for
+-- 1 + its LatestVotedRound, and increments its LatestVotedRound.  It is called RoundManager for
 -- historical reasons, because this what a previous version of LibraBFT called its main handler;
 -- this will be updated when we move towards modeling a more recent implementation.
 
-module LibraBFT.Impl.Consensus.ChainedBFT.EventProcessor
+module LibraBFT.Impl.Consensus.ChainedBFT.RoundManager
```
Comment (mark-moir) [inline LibraBFT/Impl/Consensus/ChainedBFT/RoundManager.agda:25]: This should also be moved to `LibraBFT.Impl.Consensus.RoundManager`, keeping the structure consistent with the  implementation (`ChainedBFT` is obsolete just like `EventProcessor`!).

File: LibraBFT/Impl/Consensus/ChainedBFT/RoundManager/Properties.agda [hunk 6]
```diff
@@ -16,16 +16,16 @@ open import LibraBFT.Impl.Base.Types
 open import LibraBFT.Impl.Consensus.Types
 open import LibraBFT.Impl.Util.Util
 
-module LibraBFT.Impl.Consensus.ChainedBFT.EventProcessor.Properties
```
Comment (mark-moir) [inline LibraBFT/Impl/Consensus/ChainedBFT/RoundManager/Properties.agda:19]: This should also be moved to `LibraBFT.Impl.Consensus.RoundManager.Properties`, keeping the structure consistent with the implementation.

File: LibraBFT/Yasm/Properties.agda [hunk 7]
```diff
@@ -81,7 +81,7 @@ module LibraBFT.Yasm.Properties
 
  -- TODO-1 : prove it
  postulate
-   ValidSenderForPK⇒ep≡ : ∀ {e p1 p2 α1 pk} {𝓔s : AvailableEpochs e}
```
[the remark this answers is not in the document]
Comment (haroldcarr) [reply LibraBFT/Yasm/Properties.agda:84]: done

File: LibraBFT/Impl/Properties/VotesOnceDirect.agda (hunk 1 above)
Comment (haroldcarr) [reply LibraBFT/Impl/Properties/VotesOnceDirect.agda:81]: DONE

File: LibraBFT/Impl/Properties/VotesOnceDirect.agda (hunk 2 above)
Comment (haroldcarr) [reply LibraBFT/Impl/Properties/VotesOnceDirect.agda:46]: DONE

File: LibraBFT/Impl/Consensus/Types.agda (hunk 4 above)
Comment (haroldcarr) [reply LibraBFT/Impl/Consensus/Types.agda]: DONE

File: LibraBFT/Impl/Handle.agda (hunk 3 above)
Comment (haroldcarr) [reply LibraBFT/Impl/Handle.agda]: DONE

File: LibraBFT/Impl/Consensus/ChainedBFT/RoundManager.agda (hunk 5 above)
Comment (haroldcarr) [reply LibraBFT/Impl/Consensus/ChainedBFT/RoundManager.agda]: Done.

File: LibraBFT/Impl/Consensus/ChainedBFT/RoundManager/Properties.agda (hunk 6 above)
Comment (haroldcarr) [reply LibraBFT/Impl/Consensus/ChainedBFT/RoundManager/Properties.agda]: DONE

**Comment by mark-moir** [comment]
All looks good, @haroldcarr.  Thanks for your first contribution (to this repo)! :-)

**Pull request merged**